api/src/services/email.service.ts

```diff
@@ -317,6 +317,27 @@ export class EmailService {
     );
   }
 
+  public async sendSingleUseCode(user: User, singleUseCode: string) {
+    const jurisdiction = await this.getJurisdiction(user.jurisdictions);
+    void (await this.loadTranslations(jurisdiction, user.language));
+    const emailFromAddress = await this.getEmailToSendFrom(
+      user.jurisdictions,
+      jurisdiction,
+    );
+    await this.send(
+      user.email,
+      emailFromAddress,
+      'Partners Portal account access token',
```

Comment:
Is this maybe copy pasta text?

Reply:
update! Em asked for 2 different subjects depending on if the user is a new user or a user logging in

lemme know what you think!